modules/pymol/seqalign.py

```diff
@@ -11,6 +11,25 @@
 
 from pymol import cmd, CmdException
 
+import functools
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import Bio.Align
+
+
+@functools.cache
+def _get_aligner_BLOSUM62() -> "Bio.Align.PairwiseAligner":
+    from Bio.Align import PairwiseAligner, substitution_matrices
+    blosum62 = substitution_matrices.load("BLOSUM62")
+    missing_codes = ''.join(set('JUO-.?').difference(blosum62.alphabet))
+    blosum62 = blosum62.select(blosum62.alphabet + missing_codes)
+    aligner = PairwiseAligner(internal_open_gap_score=-10,
+                              extend_gap_score=-.5,
+                              substitution_matrix=blosum62)
+    assert aligner.mode == "global"
```

Comment:
Is the intent here that we should always use `global` even if Biopython changes this as its default? If so, would a
```python
aligner.mode = "global"
```
be more future-proof?